blocks/header/headerutil.js

```diff
@@ -298,6 +298,109 @@ const processBondsType = (items) => {
   return processedData;
 };
 
+function loadStockFeed(gaTokenId) {
+  const feedURL = 'https://contentfeeds.icicidirect.com/';
```

Comment:
make this as a constant in block uti, it might change when they actually provide us the access

Reply:
agreed, will move it there.

Reply:
moved